### PR TITLE
release branch cherrypick (logger refactor leftovers)

### PR DESCRIPTION
**Pull request by dsvoronin**
bump version

### CODE REVIEW

File: subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt [hunk 1]
```diff
@@ -280,13 +281,12 @@ abstract class InHouseInstrumentationTestRunner :
...
 
-    @SuppressLint("LogNotTimber")
-    fun tryToReportUnexpectedIncident(incident: Throwable, tag: String) {
+    fun tryToReportUnexpectedIncident(incident: Throwable) {
         try {
             report.registerIncident(AppCrashException(incident))
             report.reportTestCase()
         } catch (t: Throwable) {
-            Log.w(tag, "Can't register and report unexpected incident", t)
+            logger.critical("Can't register and report unexpected incident", t)
```
Comment (eugene-krivobokov) [inline subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt]: Is it a failure for tests running?

File: subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ArgsProvider.kt [hunk 2]
```diff
@@ -49,4 +51,14 @@ class BundleArgsProvider(
             return result
         }
     }
+
+    override fun getOptionalIntArgument(name: String): Int? {
+        val defaultValue = Int.MIN_VALUE
+        val result = bundle.getInt(name, defaultValue)
+        return if (result == defaultValue) {
```
Comment (eugene-krivobokov) [inline subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ArgsProvider.kt:58]: Why don't we check it by `containsKey`?

File: subprojects/build.gradle.kts [hunk 3]
```diff
@@ -325,7 +325,7 @@ fun Project.configureBintray(vararg publications: String) {
         //   (https://github.com/bintray/gradle-bintray-plugin/issues/325)
         // - Could not upload to 'https://api.bintray.com/...':
         //   HTTP/1.1 405 Not Allowed 405 Not Allowed405 Not Allowednginx
-        override = false
+        override = true
```
Comment (eugene-krivobokov) [inline subprojects/build.gradle.kts]: Are you sure to make it default?
It can override the previous release and break all clients.

Comment (dsvoronin) [reply subprojects/build.gradle.kts]: nope, thanks

File: subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ArgsProvider.kt (hunk 2 above)
Comment (dsvoronin) [reply subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ArgsProvider.kt:58]: i think it is a bit more safe that way

from bundle.getInt
```
 catch (ClassCastException e) {
            typeWarning(key, o, "Integer", defaultValue, e);
            return defaultValue;
        }
```

and no default value means default is 0, so we will get 0 result if pass something not convertable to int

File: subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt (hunk 1 above)
Comment (dsvoronin) [reply subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt]: i reverted to warning, first look was: yes, it's absolutely critical, but then i remember all this multithread calls to onException, and that we will get "false-criticals" here

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**